### PR TITLE
[feature](alter) Add AlterDatabasePropertyStmt binlog impl

### PR DESCRIPTION
**Pull request by JackDrogon**
## Proposed changes

 Add AlterDatabasePropertyStmt binlog impl

```sql
alter database t1 set properties ("binlog.enable" = "true");
```

show
```bash
mysql> show create database t1 \G;
*************************** 1. row ***************************
       Database: t1
Create Database: CREATE DATABASE `t1`
PROPERTIES (
"binlog.enable" = "true"
)
1 row in set (0.01 sec)e" = "true"
)
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java [hunk 1]
```diff
@@ -121,6 +121,9 @@ public enum DbState {
     @SerializedName(value = "dbProperties")
     private DatabaseProperty dbProperties = new DatabaseProperty();
 
+    @SerializedName(value = "binlogConfig")
+    private BinlogConfig binlogConfig = new BinlogConfig();
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java:125]: Can it be put into `dbProperties`?

File: fe/fe-core/src/main/java/org/apache/doris/persist/AlterDatabasePropertyInfo.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import com.google.gson.annotations.SerializedName;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.Map;
+
+public class AlterDatabasePropertyInfo implements Writable {
+    @SerializedName(value = "dbName")
+    private String dbName;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/persist/AlterDatabasePropertyInfo.java:33]: Better use database id?

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java (hunk 1 above)
Comment (JackDrogon) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java:125]: I will use it frequently，so maybe cache the binlogConfig is a btter choice.

**Comment by JackDrogon** [comment]
run buildall

**Comment by JackDrogon** [comment]
Doc will be added later with other doc. Refer to PR instructions for usage

**Comment by JackDrogon** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**